libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts

```diff
@@ -71,12 +78,8 @@ export class LicenseServiceApi {
   async getGenericDrivingLicense(
     nationalId: User['nationalId'],
   ): Promise<GenericDrivingLicenseResponse[] | null> {
-    // TODO(osk) Move into env
-    const xroadDrivingLicensePath =
-      'r1/IS-DEV/GOV/10005/Logreglan-Protected/RafraentOkuskirteini-v1'
-
     const response = await this.requestApi(
-      `${xroadDrivingLicensePath}/api/Okuskirteini/${nationalId}`,
+      `${this.xroadPath}/api/Okuskirteini/${nationalId}`,
```

Comment:
For future clients, I recommend moving all xroad path logic to a lower (requestApi in this case), so this logic can call endpoints just with "/api/Okuskirteini/...".

Also rename the variable/env: xroadProviderId, and hardcode the r1 segment.

That way, ClientID and ProviderID have the same structure and (with the baseUrl) can be used to call SOAP and REST endpoints on x-road.

For REST endpoints, the request is `${xroadBaseUrl}/r1/${xroadProviderId}${endpoint}`, with client id in a header.

For SOAP endpoints, the request is `${xroadBaseUrl}` with client id AND provider id included in the SOAP request body (split up into multiple XML elements).

Reply:
Yes, makes sense, I'll add to our list for v2 to refactor this